python/grass/jupyter/utils.py

```diff
@@ -201,6 +204,23 @@ def get_rendering_size(region, width, height, default_width=600, default_height=
     return (default_width, round(default_width * region_height / region_width))
 
 
+def save_vector(name, geo_json):
+    """
+    Saves the user drawn vector.
+
+    param geojson_filename: name of the geojson file to be saved
+    param name: name with which vector should be saved
+    """
+    with tempfile.NamedTemporaryFile(
```

Comment:
I am not sure why and when to delete the temp file.

Reply:
For the why, its because it is not deleted automatically with delete=False. Take a look at the section here: https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile, it goes through the behavior of NamedTemporaryFile vs other tempfile classes.